requirements/dev.txt

```diff
@@ -123,7 +123,7 @@ idna==3.4
     #   -r requirements/ci.txt
     #   -r requirements/quality.txt
     #   requests
-importlib-metadata==4.12.0
+importlib-metadata==5.0.0
```

Comment:
No breaking change mentioned in the new version release notes. Removed some deprecated entry points. https://github.com/python/importlib_metadata/blob/main/CHANGES.rst#v500